QUESTIONS.md

```diff
@@ -3,3 +3,8 @@
 What questions can you come up with?
 
 ## Cohort Questions
+question1:
+Most web dev job posting required candidates to have a bachelor in CS. what about those of us who don not have a CS bachelor?
```

Comment:
Dont even worry about it. My last 3 jobs required bachelor degrees in the IT field.

That's the HR director writing that, if you talk to a senior developer or project manager they will only care about experience and your projects.